pkg/security/security_profile/profile/profile.go

```diff
@@ -43,6 +43,7 @@ type SecurityProfile struct {
 	loadedNano             uint64
 	selector               cgroupModel.WorkloadSelector
 	profileCookie          uint64
+	eventTypes             []model.EventType
 	anomalyDetectionEvents []model.EventType
```

Comment:
I think we should remove anomalyDetectionEvents, it don't have any purpose here anymore:
ToSecurityProfileMessage() should list profile eventTypes, no more anomalyDetectionEvents.
And it's still used by IsEventTypeValid(), which must now look at the new eventTypes instead.